src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/Target_X64/X64ReadyToRunGenericHelperNode.cs

```diff
@@ -210,6 +210,7 @@ protected sealed override void EmitCode(NodeFactory factory, ref X64Emitter enco
                 case ReadyToRunHelperId.VirtualCall:
                 case ReadyToRunHelperId.ResolveVirtualFunction:
                 case ReadyToRunHelperId.MethodEntry:
+                case ReadyToRunHelperId.InterfaceDispatchCell:
```

Comment:
I'll rename it to `VirtualDispatchCell`. The current state is confusing because there's not a 1:1 mapping between the helpers and dictionary entries. Project X reused the weirdo dictionary entries we had as workarounds on the RT side.